app/views/publish/providers/schools/new.html.erb

```diff
@@ -1,25 +1,21 @@
-<% page_title = "Add a school" %>
+<% page_title = "School details - Add School" %>
 <% content_for :page_title, title_with_error_prefix(page_title, @school_form.errors.any?) %>
-<%= content_for :before_content, render_breadcrumbs(:new_site) %>
+<%= content_for :before_content do %>
+  <%= govuk_back_link_to %> <!-- TODO: back to js search? -->
```

Comment:
This can just go back to the index page for now

Reply:
Done